cli/dcoscli/config/main.py

```diff
@@ -119,19 +188,35 @@ def _show(name):
     toml_config = config.get_config(True)
 
     if name is not None:
-        value = toml_config.get(name)
-        if value is None:
+        file_value = toml_config.get(name)
+        try:
+            # If the user presented a partial key name, eg 'core' when
+            # we have 'core.xyz'; we will get an exception here
+            effective_value, envvar_name = config.get_config_val_and_envvar(name)
```

Comment:
flake8.checker is unhappy about this, but I pretty much think it's just "wrong" in this case.
The goal of the width rule is

1 - no explosion in width, so you can reasonable read code.
2 - Don't go crazy on indentation
3 - Don't make overly complex expressions

On 1, this line is 81 characters.
On 2 we have a if -> try -> val = func(), hardly unreasonable indentation.
On 3, we have x, y = func(val); the only complexity here is the tuple unpack which is only made worse by indirection

The function name is unfortunately long which I could interperet this as fingering, but seemed the clearest way to present the distinction from the extremely similarly intended get_config_val().

Basically I don't see a better rendering of this line that is 2 characters shorter.  Is this a hard requirement in this project?